src/main/java/lotto/view/Output.java

```diff
@@ -0,0 +1,40 @@
+package lotto.view;
+
+import lotto.domain.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Output {
+    private static StringBuilder stringBuilder = new StringBuilder();
+
+    private Output() {
+    }
+
+    public static void viewPurchasedLotto(Lottos lottos) {
+        for (Lotto lotto : lottos.getLottos()) {
+            List<Integer> view = new ArrayList<>();
+            lotto.getLottoNumbers().stream()
+                    .map(number -> number.getLottoNumber())
+                    .forEach(i -> view.add(i));
```

Comment:
`메서드 참조`를 활용하면 더 편리할 것 같아요.
```suggestion
                    .map(LottoNumber::getLottoNumber)
                    .forEach(view::add);
```